documentation/tutorials/kaggle_beginner_example_classification.ipynb

```diff
@@ -0,0 +1,2105 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.    warnings.filterwarnings('ignore')
</pre><p>Is this to prevent anything in particular?  Usually warnings are helpful to users when they are interacting.</p><p> </p><p>If there's a good reason to suppress them though, that's OK.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/decision-forests/pull/128/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>